iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergSplit.java

```diff
@@ -75,7 +81,7 @@ public Path getPath() {
 
   @Override
   public byte[] getBytesForHash() {
-    Collection<FileScanTask> fileScanTasks = innerSplit.task().files();
+    Collection<FileScanTask> fileScanTasks = innerSplit.task().tasks();
```

Comment:
Because only `CombinedScanTask` has `tasks`. Renamed `IcebergSplit#task`
https://github.com/apache/hive/pull/5409/commits/3c57a67327340197c2f1798503ab1927a5104cdd